assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/TicketAssignmentsManager.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import ErrorMessage from './ErrorMessage';
+import { Table } from './table';
+import { FilterBar } from '../filters';
+import { useDatesAndTickets } from '../data';
+import { useDataState } from '../data';
+
+import './style.scss';
+
+const TicketAssignmentsManager: React.FC = () => {
+	const datesAndTickets = useDatesAndTickets();
+	const dataState = useDataState();
+
+	return (
+		<div id='ticket-assignment-manager'>
+			<FilterBar />
+			<ErrorMessage dataState={dataState} />
```

Comment:
a little bit of spacing would be nice here:
![image](https://user-images.githubusercontent.com/1477453/77060435-3b335780-69e1-11ea-978d-e33df8abd015.png)

Reply:
Yeah, styling is not an issue for now. We can deal with it later.

Reply:
Added padding